pkgs/development/haskell-modules/configuration-common.nix

```
@@ -2572,6 +2578,11 @@ self: super: {
     lsp = self.lsp_1_5_0_0;
   };
 
+  vector-hashtables = unmarkBroken (super.vector-hashtables.override {
+    hspec = self.hspec_2_7_10;
+    hspec-discover = self.hspec-discover_2_7_10;
+  });
```

The better solution is to use the default `hspec` version we use and relaxing the version bound which turns out to be unnecessary in this case:

```suggestion
  # Too strict bounds on hspec
  # https://github.com/klapaucius/vector-hashtables/issues/11
  vector-hashtables = doJailbreak super.vector-hashtables;
```